modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -165,6 +172,127 @@ inline void copyFromIE(const IE::Blob::Ptr &blob, MatType &mat) {
     }
 }
 
+#if INF_ENGINE_RELEASE < 2020000000  // < 2020.1
+// Load extensions (taken from DNN module)
+std::vector<std::string> getExtensions(const cv::gapi::ie::detail::ParamDesc& params) {
+    std::vector<std::string> candidates;
+    if (params.device_id == "CPU" || params.device_id == "FPGA")
+    {
+        const std::string suffixes[] = { "_avx2", "_sse4", ""};
+        const bool haveFeature[] = {
+            cv::checkHardwareSupport(CPU_AVX2),
+            cv::checkHardwareSupport(CPU_SSE4_2),
+            true
+        };
```

Comment:
This is old logic related to the releases with multiple compiled `cpu_extension` in OpenVINO - no there is no extensions so you don't need to load them by suffix.

Reply:
@dkurt OK, but this part of code had been already there, I just moved it to a separate function; do we need it for the older OV releases? If I understand correctly, it was relevant for 2019.R3 when @dmatveev added this extensions loading to allow detectors work safely with "CPU" target

Reply:
Yes, but if only G-API has plans to support 2019R3

Reply:
@dkurt does DNN module plan to support that?

Reply:
@dkurt It's not really a new functionality as it's already been implemented here; I just wrapped it into a separate function
@dmatveev If DNN module keeps it, I think we should too